src/phoenix/server/api/schema.py

```diff
@@ -97,8 +97,14 @@ def node(self, id: GlobalID, info: Info[Context, None]) -> Node:
             embedding_dimension = info.context.model.embedding_dimensions[node_id]
             return to_gql_embedding_dimension(node_id, embedding_dimension)
         elif type_name == "Project":
-            # TODO: implement ID
-            ...
+            if (traces := info.context.traces) is not None:
+                projects_list = [proj for proj in traces.get_projects()]
+                project_item = projects_list[node_id]
+                if project_item is not None:
```

Comment:
There could be an index-out-of-bound error with the list approach here. Use dict instead.
```suggestion
                projects = dict(enumerate(traces.get_projects()))                
                if (project_item := projects.get(node_id)) is not None:
```

Reply:
oh gotcha